File: src/main/java/com/lcaohoanq/nocket/domain/asset/IFileStoreService.java
package com.lcaohoanq.nocket.domain.asset;

import java.io.IOException;
import java.util.List;
import org.springframework.web.multipart.MultipartFile;

public interface IFileStoreService {

    String storeFile(MultipartFile file) throws IOException;
    
    MultipartFile validateProductImage(MultipartFile file) throws IOException;
}

File: server/src/main/kotlin/com/lcaohoanq/nocket/domain/asset/IFileStoreService.kt
package com.lcaohoanq.nocket.domain.asset

import org.springframework.web.multipart.MultipartFile
import java.io.IOException

interface IFileStoreService {
    @Throws(IOException::class)
    fun storeFile(file: MultipartFile): String

    @Throws(IOException::class)
    fun validateProductImage(file: MultipartFile): MultipartFile
}
